### PR TITLE
Report iframe activation mismatches for resize

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-ad/0.1/amp-ad-ui.js [hunk 1]
```diff
@@ -195,11 +197,22 @@ export class AmpAdUIHandler {
...
       return Promise.resolve(resizeInfo);
     }
+    const performance = Services.performanceForOrNull(this.baseInstance_.win);
+    const activated =
+      event && event.userActivation && event.userActivation.hasBeenActive;
     return this.baseInstance_.attemptChangeSize(newHeight, newWidth).then(
       () => {
+        if (performance) {
+          // Report false positives.
+          performance.tickDelta('rsfp', activated ? 1 : 1000);
```
Comment (lannka) [inline extensions/amp-ad/0.1/amp-ad-ui.js]: why 1 & 1000?
for simplicity, shall we just use user error?

Comment (dvoytenko) [reply extensions/amp-ad/0.1/amp-ad-ui.js]: Just expanding the reporting range. Really any numbers would do. User error however seems wrong and distracting and we don't really have good way of charting them, like we do with latency-style dashboards.

**Comment by dvoytenko** [comment]
@lannka I just wanted to get a very preliminary data on where we disallowed resize even though the was activation. So I think "false negative" is fair. I agree that "false positive" is a bit odd. I only included it for control/comparison since I have no idea how often we "reject".

**Comment by lannka** [comment]
I guess what's most interesting will be:
1) how many resize in total
2) how many get rejected
3) how many get rejected but there was a user interaction?

I know those can be calculated from what you're collecting. But shall we just be direct? Just so to not confuse people about those "false positive" "false negative" definition.

Also, I don't really see the need for a percentile chart for those numbers. We just need 3 raw numbers, which can be easily seem in our error reporting. No CSI pipeline changes needed, no 2-day reporting delays (in fact our CSI pipeline is having issues at this moment go/amp-csi ).

**Pull request merged**